diff -ur discount-2.0.9/Makefile.in discount-2.0.9.new/Makefile.in
--- discount-2.0.9/Makefile.in	2011-05-11 01:33:30.868994630 +0900
+++ discount-2.0.9.new/Makefile.in	2011-05-11 01:39:09.048993059 +0900
@@ -22,9 +22,12 @@
 
 all: $(PGMS) $(SAMPLE_PGMS) $(TESTFRAMEWORK)
 
-install: $(PGMS)
+install: $(PGMS) install.man
+	@INSTALL_DIR@ $(DESTDIR)/$(BINDIR)
 	@INSTALL_PROGRAM@ $(PGMS) $(DESTDIR)/$(BINDIR)
+	@INSTALL_DIR@ $(DESTDIR)/$(LIBDIR)
 	./librarian.sh install libmarkdown VERSION $(DESTDIR)/$(LIBDIR)
+	@INSTALL_DIR@ $(DESTDIR)/$(INCDIR)
 	@INSTALL_DATA@ mkdio.h $(DESTDIR)/$(INCDIR)
 
 install.everything: install install.samples install.man
diff -ur discount-2.0.9/configure.inc discount-2.0.9.new/configure.inc
--- discount-2.0.9/configure.inc	2011-05-11 01:33:30.902997106 +0900
+++ discount-2.0.9.new/configure.inc	2011-05-11 01:30:39.676956924 +0900
@@ -1240,7 +1240,6 @@
 install)$PROG_INSTALL -c \$FULLNAME "\$1"
 	$PROG_LN_S -f \$FULLNAME \$1/\$LIBNAME.\$MAJOR
 	$PROG_LN_S -f \$FULLNAME \$1/\$LIBNAME
-	test "$LDCONFIG" && $LDCONFIG "\$1"
 	;;
 esac
 EOF
